### DISCUSSION TITLE
Naming policy for boolean options

### DISCUSSION THREAD

**Discussion by ST-DDT** [faker-js/faker#806] [Community-Polls]
I would like to collect some input on how we should name our boolean options:

- `allowSpecialCharacters` or `specialCharacters`
- `allowLeadingZero` or `leadingZero`

Any preference or other suggestions?

Please vote by upvoting the individual answers (arrow up)

**Comment by prisis** [comment]
Add `allow` for these

**Comment by ST-DDT** [comment]
Omit `allow` for boolean options
